.gitignore

```diff
@@ -42,6 +42,7 @@ Temporary Items
 /docker/
 /.direnv/
 /testdata/
+/example/
```

Comment:
I'm not sure what kind of example you have locally but have you considered adding it to the `examples` folder and committing it?

Reply:
Good point, just a left over, removing it

Reply:
Was just thinking of adding examples of how to use it, but will keep it for a separate PR